Add auto-generated dbt model and macro docs (#264)
snowplow/documentation: docs/modeling-your-data/modeling-your-data-with-dbt/reference/index.md

---
title: "Reference"
sidebar_position: 9999
description: Reference information relating to dbt models and macros
---

:::info

The following pages are reference documents intended for developers and advanced users of the packages. 

> These pages are based on data generated by dbt, which is sometimes incomplete. For example, when a variable enabled a column it may not be present in the column list. Diabled models are more likely to be incomplete as they do not generate model dependencies, this issue mostly exists for redshift where additional tables are used.

Reviewer: ```suggestion
These pages are based on data generated by dbt, which is sometimes incomplete. For example, a column may be missing from the column list if it’s enabled by a variable. Disabled models are more likely to be incomplete as they do not generate model dependencies — this issue mostly exists for Redshift where additional tables are used.
```
One of the few cases when passive voice makes things clearer!
